scripts/process_notebooks.py

```diff
@@ -114,15 +114,15 @@ def should_process(path):
     if errors or args.check_only:
         exit(errors)
 
-    # Further filter the notebooks to run post-processing only on tutorials
-    tutorials = {
+    # Further filter the notebooks to run post-processing only on tutorials and projects
```

Comment:
I think the only other kind of notebook would be the style guide notebook. We probably want that to get the same cleaning / badge redirecting as the other notebooks (its badge is chronically broken). So now that we have conditional logic to limit exercise post-processing to tutorials, I think we can drop this step and just loop over the notebooks dict.

Reply:
The style-guide notebook is now living in `content-creation` repo. But yes, I agree that we have to apply the process to any notebook in the repo, unless otherwise specified, as the colab badge link is not updated and thus, points in a branch other that `main`. But for nma-cn, `course-content` repo, this makes sense.

So, we can remove this:
https://github.com/NeuromatchAcademy/nmaci/blob/postprocess/scripts/process_notebooks.py#L117-L122

and run the for loop directly in notebooks.

`for nb_path, nb in notebooks.items()`

Is this what you mean?

Reply:
Yes, precisely.

Reply:
Now the process will run on all notebooks in the repo. Solutions, static, student directories and notebooks only if notebook lives under `tutorials/`. So, we have to submit only run from top to bottom notebooks